src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolCaseMappingTest.java

```diff
@@ -297,13 +304,19 @@ void shouldReturnDoubleCaseAllocatorRolesForNewRule() {
 
         //assertion
         assertFalse(roleAssignments.isEmpty());
-        assertEquals(6,roleAssignments.size());
+        assertEquals(8,roleAssignments.size());
         assertEquals("senior-tribunal-caseworker",roleAssignments.get(0).getRoleName());
         assertEquals("senior-tribunal-caseworker",roleAssignments.get(1).getRoleName());
         assertEquals("tribunal-caseworker",roleAssignments.get(2).getRoleName());
         assertEquals("tribunal-caseworker",roleAssignments.get(3).getRoleName());
         assertEquals("case-allocator",roleAssignments.get(4).getRoleName());
+        assertEquals(RoleCategory.LEGAL_OPERATIONS,roleAssignments.get(4).getRoleCategory());
```

Comment:
why we are creating  duplicate roles here?

Reply:
These are existing test, modified/added new assert to match new rules